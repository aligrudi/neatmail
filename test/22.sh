# neatmail test
./mail pn -h From: -h Subject: <$1 >/tmp/.nmtest.box 2>/dev/null
cat /tmp/.nmtest.box
rm /tmp/.nmtest.box
