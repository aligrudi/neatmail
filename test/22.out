From root Tue Dec 21 01:01:01 2015
From: A <a@a.eu>
Subject: Message from A

This is A.

From root Tue Dec 22 02:02:02 2015
From: B <b@b.eu>
Subject: Message from B

This is B.

From root Tue Dec 23 03:03:03 2015
From: C <c@c.eu>
Subject: Message from C

This is C.

